src/core/shape/attributes.js

```diff
@@ -71,26 +98,75 @@ p5.prototype.ellipseMode = function(m) {
 };
 
 /**
- * Draws all geometry with jagged (aliased) edges.
+ * Draws certain features with jagged (aliased) edges.
```

Comment:
Could you share the reason to make the change from 'Draws all geometry' to 'Draws certain features'? It seems make more sense to 'draw geometry' than 'draw features'. If it's important to keep 'certain', how about 'Draws certain geometry'? @nickmcintyre

Reply:
I think the reason for this is that there's no way to turn off antialiasing on lines and shapes in 2D mode, and I think this confuses a lot of users (it's come up in the discord a few times.) In 2D mode, I believe this just affects image scaling in 2D mode, so it actually doesn't affect geometry at all, only images. Somewhat confusingly, this *does not* affect image scaling in WebGL mode (you have to use an [undocumented method on p5.Texture](https://github.com/processing/p5.js/blob/7a3d73bba4214dab2319e18c8efa4aaa5db3d010/src/webgl/p5.Texture.js#L337)) but *does* actually affect antialiasing. The situation is unfortunately a bit complicated right now :')

I think the description in this PR does a pretty good job of mentioning the key details quickly.

Reply:
@Qianqianye basically what @davepagurek said. I sketched a few test cases based on #5472 and tried to describe what I saw.